store/store.go

```diff
@@ -146,6 +152,13 @@ func myappsURL() string {
 	return "https://myapps.developer.ubuntu.com/api/2.0"
 }
 
+func scaURL() string {
+	if useStagingStore() {
```

Comment:
this uses `SNAPPY_USE_STAGING_CPI` to return a myapps url. Are you sure you don't want `SNAPPY_USE_STAGING_MYAPPS`?